extension/src/providers/gotoProvider.ts

```diff
@@ -30,7 +27,10 @@ export class AutolispDefinitionProvider implements vscode.DefinitionProvider{
 			const locations: vscode.Location[] = [];
 			if (isFunction) {
 				// This has a "preference" for opened and project documents for actual definitions, but will only handle variables on the opened document.
-				let possible = this.findDefunMatches(selected, AutoLispExt.Documents.OpenedDocuments.concat(AutoLispExt.Documents.ProjectDocuments));
+				let possible = this.findDefunMatches(selected, [AutoLispExt.Documents.ActiveDocument]);
+				if (possible.length === 0) {
+					possible = this.findDefunMatches(selected, AutoLispExt.Documents.OpenedDocuments.concat(AutoLispExt.Documents.ProjectDocuments));
```

Comment:
`ActiveDocument` may be included in `OpenedDocuments` or `ProjectDocuments`, and maybe  `OpenedDocuments` or `ProjectDocuments` has duplicate documents. In the latter PR it looks like we can filter those duplicate files, save the time to search in the atoms forest.